BoxContentSDK/BoxContentSDK/Requests/BOXFolderRequest.m

```diff
@@ -3,6 +3,7 @@
 //  BoxContentSDK
 //
 
+#import "BOXContentClient_Private.h"
```

Comment:
We should think twice before doing something like this.

It's not good for a class to work it's way upwards. I'm not sure we want to create this dependency. The request shouldn't have to know about the content client...
Is this really unavoidable?